android/envoy/src/main/kotlin/org/greatfire/envoy/EnvoyConnectWorker.kt

```diff
@@ -52,8 +52,12 @@ class EnvoyConnectWorker(
                 // XXX ask for more URLs?
                 break
             }  else if (state.connected.get()) {
-                // Log.d(WTAG, "ALREADY CONNECTED, BREAK")
-                break
+                if (state.debugMode) {
+                    // Log.d(WTAG, "ALREADY CONNECTED (debugging), CONTINUE")
+                } else {
+                    // Log.d(WTAG, "ALREADY CONNECTED (not debugging), BREAK")
```

Comment:
is there any reason to save all these commented out Log calls?

Reply:
removed the logs and the empty sections, converted some of them into comments for clarity.